.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,71 @@
+name: CodeQL Analysis
+
+on:
+  push:
+  pull_request:
+  schedule:
+    - cron: '0 8 * * *'
+
+jobs:
+  analyze:
+    name: codeql-analysis
+    runs-on: windows-latest
+    strategy:
+      matrix:
+        configuration: [ Debug, Release ]
```

Comment:
Is it necessary to do this for both configurations?? I would suggest to change configuration value to `Production` only, both `Debug` and `Release` configuration are used for internal builds, public releases are built with `Production` configuration.